.github/workflows/scheduled.yml

```
@@ -218,6 +218,40 @@ jobs:
       - name: Deploy to GitHub Pages
         id: deployment
         uses: actions/deploy-pages@v1
+  report-status:
+    name: Report deploy status
+    needs: [generate-cask, generate-core, generate-analytics, generate-samples, build, deploy]
+    if: ${{ always() && github.ref_name == 'master' }}
+    env:
+      GH_TOKEN: ${{ secrets.HOMEBREW_GITHUB_PUBLIC_REPO_TOKEN }}
+      REPO: ${{ github.repository }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Detect existing deploy failure issue
+        run: |
+          EXISTING_ISSUE=$(gh issue list --repo "$REPO" --author BrewTestBot --label 'deploy failure' | awk '{print $1}')
```

What if there's more than one?

---

Yeah, I mean that's a potential issue. I decided that since this is just intended to be a developer tool, it's not super important to get every edge case. But if you think it's a significant enough issue I can work on improving the behavior in that situation.

Maybe it's worth doing this to ensure that it won't fail or do something totally unpredictable?

```suggestion
          EXISTING_ISSUE=$(gh issue list --repo "$REPO" --author BrewTestBot --label 'deploy failure' | awk '{print $1}' | head -n 1)
```

---

Nah, fine with me if you think this will basically never happen.